src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -182,16 +195,49 @@ export const CommentTeaser = ({
             </Link>
           </CommentBodyParagraph>
         </div>
+
+        {documentPreview && (
```

Comment:
Question: This changes all comment teasers to add an image if they have a twitter image. But in an element like the discussion section on the main feed, we wouldn't want images there. Should there be a prop "withPreview"? Or does the query there not ask for "document meta twitterImage" and thus it's ok to not use a prop?
![Screenshot 2020-12-04 at 11 08 14](https://user-images.githubusercontent.com/20746301/101150787-03c51380-3621-11eb-9fc3-8f3fcf1fe10d.jpg)

Reply:
You make a good point. Using the twitter image implicitly is weird.